README.md

```diff
@@ -1,4 +1,7 @@
-# [ScummVM README](https://www.scummvm.org/) · [![Build Status](https://travis-ci.org/scummvm/scummvm.svg?branch=master)](https://travis-ci.org/scummvm/scummvm) ![CI](https://github.com/scummvm/scummvm/workflows/CI/badge.svg) [![Translation status](https://translations.scummvm.org/widgets/scummvm/-/scummvm/svg-badge.svg)](https://translations.scummvm.org/engage/scummvm/?utm_source=widget) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](CONTRIBUTING.md#pull-requests) [![Codacy Badge](https://app.codacy.com/project/badge/Grade/e06e5b18f8464fef859b5a7f78d10357)](https://www.codacy.com/gh/scummvm/scummvm/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=scummvm/scummvm&amp;utm_campaign=Badge_Grade)
+
+[<img align = "center" alt = "ScummVm" width ="500px" src = "https://user-images.githubusercontent.com/61404154/104793881-5b32d380-57ca-11eb-8dc4-c516db556450.png" />](https://www.scummvm.org/)
```

Comment:
Please fix formatting and use: https://raw.githubusercontent.com/scummvm/scummvm-media/master/originals/scummvm_logo.png instead of a randomly uploaded user image?

Reply:
Also not sure the align attribute is doing anything or necessary